src/ciflow-bot.ts

```diff
@@ -20,7 +20,16 @@ export class CIFlowBot {
   static readonly event_issue_comment = 'issue_comment';
   static readonly event_pull_request = 'pull_request';
   static readonly pr_label_prefix = 'ciflow/';
-  static readonly rollout_users = ['zhouzhuojie']; // slow rollout to specific group of users first
+  static readonly rollout_users = [
+    'driazati',
+    'janeyx99',
+    'malfet',
+    'samestep',
+    'seemethere',
+    'walterddr',
+    'zhouzhuojie'
+  ]; // slow rollout to specific group of users first
+
```

Comment:
@pytorch/pytorch-dev-infra hey team, let me know you want to include more folks to the rollout. I also added wiki page about ciflow https://github.com/pytorch/pytorch/wiki/Continuous-Integration#using-ciflow